modular_skyrat/code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -2,7 +2,7 @@
 	song = sound('modular_skyrat/sound/ambience/theopenedway.ogg', 100) //Shadow of the colossus OST
 	songlength = 1170
 
-/mob/living/simple_animal/hostile/megafauna/colossus/proc/enrage(mob/living/L)
+/mob/living/simple_animal/hostile/megafauna/colossus/enrage(mob/living/L)
```

Comment:
Is this supposed to be changing?

Reply:
this was causing compile errors, i think it isn't changed anymore since i merged upstream to this